### ISSUE TITLE
[Accessibility] Support AltText for AdaptiveTextBlock

### ISSUE THREAD

**Issue by nimercer** [microsoft/AdaptiveCards#4284]
# Platform

* .NET HTML
* .NET WPF
* Android
* iOS
* JavaScript
* UWP

# Author or host

Author & Host

If you're an author, who are you sending cards to?
1.2.9

Explain your issue. If you just have a question, please post [on StackOverflow instead]

We've been seeing some accessibility issues where the information written in an Adaptive Text Block is not enough information for a narrator user or narrator reads it out strangely. Some examples below

- _1hr_ is read out as _1 H R_ (rather than the expected '_One Hour'_)
- Any text block with a bullet point or unexpected character cannot be read out by narrator so causes accessibility issues.

It would be great if Adaptive could support an altText parameter for Adaptive text blocks (to be read out by screen readers) as part of the schema, that then is used an the screen reader text. 

Something like the below

```
        {
            "type": "TextBlock",
            "text": "1hr",
            "altText": "One hour"
        }
```

**Comment by shalinijoshi19** [comment]
@paulcam206 / @RebeccaAnne i know we'd been talking about this as part of some existing Accessibility bug fixes; Do we have an issue that had been tracking this already? Btw will the work on Labels for inputs try to help here at all or not really?

**Comment by RebeccaAnne** [comment]
> @paulcam206 / @RebeccaAnne i know we'd been talking about this as part of some existing Accessibility bug fixes; Do we have an issue that had been tracking this already? Btw will the work on Labels for inputs try to help here at all or not really?

@shalinijoshi19 I don't know of a github issue tracking this explicitly, but we also have one of the VSO bugs that would benefit from this change. The inputs work doesn't affect this case, this request is for text blocks.

**Comment by paulcam206** [comment]
^^^ what @RebeccaAnne said :)
